src/main/java/org/example/homework10/task2/Employee.java

```diff
@@ -0,0 +1,17 @@
+package org.example.homework10.task2;
+
+public record Employee(String firstName,
+                       String lastName,
+                       String hireDate) {
+    public String firstName() {
+        return firstName;
+    }
+
+    public String lastName() {
+        return lastName;
+    }
+
+    public String hireDate() {
```

Comment:
you don't need this getter in the Record

Reply:
removed